gatekeeper/client/client.go

```diff
@@ -29,20 +32,46 @@ type Client struct {
 }
 
 // NewClient returns a new client to a Gatekeeper host that can bootstrap this machine
-func NewClient(host string) *Client {
+func NewClient(host, caFile string) (*Client, error) {
 	if !strings.HasSuffix(host, "/") {
 		host += "/"
 	}
+
+	caPool, err := x509.SystemCertPool()
+	if err != nil {
```

Comment:
For testing and development purposes, it makes sense for the Gatekeeper to not require SSL. However, in all other cases SSL should be required.

So, perhaps, we just add a log to create noise (a WARN) about the fact that SSL is not enabled.

Regarding the error, in the case where `--ca` is not provided *and* we can't get the SystemCertPool, we'd end up swallowing the error. Instead, I think this should be fatal instead of silently swallowed with an empty cert pool. Perhaps I'm missing some context?